lib/Controller/Config/ControllerDataProvider.php

```diff
@@ -232,6 +232,11 @@ public function getTemplates(): array
             $templates = array_merge($templates, $this->findTemplates($appPath));
         }
 
+        $symfonyPath = realpath(implode(DIRECTORY_SEPARATOR, [PIMCORE_PROJECT_ROOT, 'templates']));
+        if ($symfonyPath && file_exists($symfonyPath) && is_dir($symfonyPath)) {
```

Comment:
I doubt that checking for both, `file_exists` & `is_dir` is really necessary.

```suggestion
        if ($symfonyPath && is_dir($symfonyPath)) {
```

Reply:
That's correct, `is_dir(null)` will return a `false` boolean. I just copied the above and below checks, so I'll change those as well.